src/__tests__/fields/contact-field.spec.ts

```diff
@@ -68,6 +69,82 @@ describe("contact-field", () => {
 		});
 	});
 
+	describe("fixed country number", () => {
+		it("should validate specific country", () => {
```

Comment:
- remove `fixed country number` test suite
- move this test to under `international numbers` test suite
- rename description as `should validate against specific country when specified`